.gitignore

```diff
@@ -0,0 +1,5 @@
+.env
```

Comment:
You should add `__pycache__` folders and youre `.idea` so it won't pollute the repository with uneccessary files.